adr/00008-handle-missing-env-vars.md

```diff
@@ -0,0 +1,33 @@
+# 8. handle-missing-env-vars
```

Comment:
Discussed offline, but this change shouldn't require an ADR as it's quite small :+1:

Reply:
Removed